templates/openwisp2/settings.py

```diff
@@ -110,12 +116,28 @@
     'pipeline.middleware.MinifyHTMLMiddleware'
 ]
 
+{% if openwisp2_radius %}
+OPENWISP_RADIUS_FREERADIUS_ALLOWED_HOSTS = {{ openwisp2_radius_allowed_hosts }}
+
+# SMS
+REST_AUTH_SERIALIZERS = {
+    'PASSWORD_RESET_SERIALIZER': 'openwisp_radius.api.serializers.PasswordResetSerializer',
+}
+REST_AUTH_REGISTER_SERIALIZERS = {
+    'REGISTER_SERIALIZER': 'openwisp_radius.api.serializers.RegisterSerializer',
+}
+OPENWISP_RADIUS_SMS_TOKEN_MAX_IP_DAILY = {{ openwisp2_radius_sms_token_max_ip_daily }}
+SENDSMS_BACKEND = '{{ openwisp2_radius_sms_backend }}'
+
+OPENWISP_USERS_AUTH_API = {{ openwisp2_users_auth_api }}
```

Comment:
this one can be moved outside of this if since it can be used also by other modules.